tests/smart/modules/test_powerprotection.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+from pytest_mock import MockerFixture
+
+from kasa import Module, SmartDevice
+from kasa.smart.modules import PowerProtection
+
+from ...device_fixtures import parametrize
+
+powerprotection = parametrize(
+    "has powerprotection",
+    component_filter="power_protection",
+    protocol_filter={"SMART"},
+)
+
+
+def _skip_on_unavailable(dev: SmartDevice):
+    if Module.PowerProtection not in dev.modules:
+        pytest.skip(f"No powerprotection module on {dev}, maybe a strip parent?")
```

Comment:
```suggestion
```
Usually this scenario is handled by using the helper method [`get_parent_and_child_modules`](https://github.com/python-kasa/python-kasa/blob/6d9b4421feffd32e0e899df38d154c2c6d68178d/tests/device_fixtures.py#L533) in `device_fixtures`. i.e.
```py
powerprot = next(get_parent_and_child_modules(dev, Module.PowerProtection))
```
You could test all the children if you think it's necessary or just test one with `next`

Reply:
Will do.